notification-server/.env.template

```diff
@@ -0,0 +1,2 @@
+PGPASSWORD=$DB_PASSWORD
```

Comment:
can we add these two services to a github action to ensure they compile.

Reply:
Sorry I do not understand

Reply:
This file is alone useless and only serves as a template for envsubst.

Reply:
The comment has nothing to do with the file or line in the file. I meant to add a general comment, that we don't have a github action currently that compiles your 2 services as part of our CI flow.

Reply:
Cargo clippy does compile it?

Reply:
I do not see why I have to deviate from the rest of the services in misc

Reply:
I think this should be fine, as it is included in the matrix for the jobs in `.github/workflows/ci.yaml`

Reply:
True, see it now.